components/renku_data_services/message_queue/avro_models/io/renku/events/v1/project_updated.py

```diff
@@ -19,6 +19,7 @@ class ProjectUpdated(AvroModel):
     repositories: List[str]
     visibility: Visibility
     description: Optional[str]
+    keywords: List[str]
```

Comment:
```suggestion
    keywords: List[str] = field(default_factory=list)
```

But also you have to import field from dataclasses above.

Because the avro schema says that the default value of the field should be `[]` but the code generation tool misses this.

Reply:
This gets overwritten we update the avro schemas (via `make update_avro`) and we need to make this change every time. Is this what you mean? I already take care of this case in the code and send an empty list when `keywords` is `null`.

Reply:
Yes sorry. Lets keep it as is. I forgot it was part of the auto-generated code.